package.json

```diff
@@ -34,6 +34,7 @@
     "hast-util-to-string": "3.0.1",
     "iframe-resizer-react": "^1.1.0",
     "leaflet": "^1.9.4",
+    "lucide-react": "^0.468.0",
```

Comment:
forgive me; we have too many icon packages. Will figure out cleaning all of those....